features/conversation-list/hooks/use-delete-dm.ts

```diff
@@ -42,26 +42,23 @@ export const useDeleteDm = (dm: DmWithCodecsType) => {
         topic: dm.topic,
       }),
     onMutate: () => {
-      const previousIsDeleted = getConversationDataQueryData({
+      const previousIsDeleted = getConversationMetadataQueryData({
         account: currentAccount,
         topic: dm.topic,
-        context: "useDeleteDm",
       })?.isDeleted;
 
-      updateConversationDataQueryData({
+      updateConversationMetadataQueryData({
         account: currentAccount,
         topic: dm.topic,
-        context: "useDeleteDm",
```

Comment:
I thoubght you added this context pattern @thierryskoda why are you removing now?